modules/jolt_physics/jolt_physics_server_3d.cpp

```diff
@@ -93,7 +94,14 @@ RID JoltPhysicsServer3D::box_shape_create() {
 }
 
 RID JoltPhysicsServer3D::capsule_shape_create() {
-	JoltShape3D *shape = memnew(JoltCapsuleShape3D);
+	JoltShape3D *shape = memnew(JoltTaperedCapsuleShape3D);
```

Comment:
Maybe I don't fully understand this change, but a CapsuleShape is cheaper than a TaperedCapsuleShape. So if someone is requesting a capsule shape, I would provide a capsule shape and not a tapered one.

Reply:
Will revise